src/frontendproxy/envoy.tmpl.yaml

```diff
@@ -57,14 +59,26 @@ static_resources:
           explicit_http_config:
             http2_protocol_options: {}
       load_assignment:
-        cluster_name: opentelemetry_collector
+        cluster_name: opentelemetry_collector_grpc
         endpoints:
           - lb_endpoints:
               - endpoint:
                   address:
                     socket_address:
                       address: ${OTEL_COLLECTOR_HOST}
-                      port_value: ${OTEL_COLLECTOR_PORT}
+                      port_value: ${OTEL_COLLECTOR_PORT_GRPC}
+    - name: opentelemetry_collector_http
```

Comment:
I'm very new to Envoy, so I'm unsure if we need a separate cluster for gRPC and HTTP

Reply:
I'm not an Envoy expert either, but my understanding is you need multiple clusters to make this happen.